test/WebJobs.Script.Tests/ScriptHostTests.cs

```diff
@@ -47,6 +47,11 @@ public static void GetRelativeDirectory_ReturnsExpectedDirectoryName(string path
             ScriptHost host = _fixture.Host;
             string debugSentinelFilePath = Path.Combine(host.ScriptConfig.RootLogPath, "Host", ScriptConstants.DebugSentinelFileName);
 
+            if (!File.Exists(debugSentinelFilePath))
```

Comment:
Unrelated to this change, but a small bug fix with this test. This was expecting the file to be there (it works because other tests create this file and leave it behind) so it would fail when executed in isolation.